src/Avalonia.Visuals/Media/HsvColor.cs

```diff
@@ -0,0 +1,493 @@
+﻿using System;
+
+namespace Avalonia.Media
+{
+    /// <summary>
+    /// Defines a color using the hue/saturation/value (HSV) model.
+    /// </summary>
+    public struct HsvColor : IEquatable<HsvColor>
+    {
+        /// <summary>
+        /// Represents a null <see cref="HsvColor"/> with zero set for all channels.
+        /// </summary>
+        public static readonly HsvColor Empty = new HsvColor();
+
+        ////////////////////////////////////////////////////////////////////////
```

Comment:
please avoid using these style of comments. use /* */ instead.

Reply:
Has that been documented and agreed for the Avalonia code-base? Is there a format guide?

The issue with /* */ comments is that they block larger /* */ comments. If you ever need to comment-out code at least temporarily, /* */ quickly gets in the way. For that reason, the internal convention is to use // for all comments that should remain in code and /* */ is reserved for code-blocks that are temporarily disabled.

Reply:
It's just convention. I just havent seen this kind of comment style being committed in all my years here.

Reply:
Yea, section headings aren't widely used. It is extremely useful to keep code organized so each dev doesn't just puts code wherever they like when they modify the file. Putting similar code together improves readability and maintainability quite a bit. I think this is used more in the embedded world where the requirements are far more strict and things like AUTOSAR and detailed code checking, design reviews and proofs mean perfect code is topmost priority. Over the air updates didn't always exist... I digress.

Since HsvColor will likely get no major changes in its life, I can remove these headings if you really don't like them. It would be better than using `/* */` which would also be strange for you but have the added concern of making it difficult to disable code in the future.

Reply:
Well, #regions are different than this. They are frowned-upon these days for other reasons.

If everyone is against these regions I will remove them. I'm surprised no one wants to enforce the structure of code within objects a bit more though -- it is standard in some projects.

Reply:
Removed due to complaints :)